tool/generator/lib/src/projection/utils.dart

```diff
@@ -160,14 +160,34 @@ String importForWin32Type(TypeIdentifier identifier) {
   }
 }
 
-/// Converts a namespace (e.g. `Windows.Win32.System.Console`) and returns the
-/// matching folder (e.g. `system/console`).
+/// Converts a namespace (e.g. `Windows.Win32.System.Console.CONSOLE_FONT_INFO`)
+/// and returns the matching folder (e.g. `system/console`).
 String folderFromNamespace(String namespace) {
   final segments = namespace.split('.').skip(2).toList()..removeLast();
 
   return segments.join('/').toLowerCase();
 }
 
+/// Converts a namespace (e.g. `Windows.Storage.Pickers.FileOpenPicker`) and
+/// returns the matching folder (e.g. `storage/pickers`).
+String folderFromWinRTNamespace(String namespace) {
```

Comment:
As with `folderFromNamespace`, we should rename this if it's designed for types. https://docs.microsoft.com/en-us/uwp/winrt-cref/winmd-files#namespace

Reply:
Yeah, it's designed for types. I will rename this and the below function.